src/base/bittorrent/session.h

```diff
@@ -257,6 +257,8 @@ namespace BitTorrent
         virtual void setPerformanceWarningEnabled(bool enable) = 0;
         virtual int saveResumeDataInterval() const = 0;
         virtual void setSaveResumeDataInterval(int value) = 0;
+        virtual int saveStatisticsInterval() const = 0;
+        virtual void setSaveStatisticsInterval(int value) = 0;
```

Comment:
```suggestion
        virtual std::chrono::minutes saveStatisticsInterval() const = 0;
        virtual void setSaveStatisticsInterval(std::chrono::minutes value) = 0;
```
Take a look at Preferences::getWebUIBanDuration, Preferences::setWebUIBanDuration for an example.

Reply:
IMO it's better to use int, as UI elements and class storage would also be int.  std::chrono::minutes is only needed when checking the timer, reverse conversion not even needed in this case. Might be wrong since I'm not familiar with the codebase.

Will update the pull req with the changes in the following hours

Reply:
I agree that it is generally better to use `std::chrono` over int when referring to time interval, but it still feels a bit _forced_ here.  The GUI converts the `int` value of the spinbox to `std::chrono::minutes` before calling `session->setSaveStatisticsInterval(...)`, which in turn casts it back to `int` to store it. 

I'll update the pull request in a few minutes and you can take a look.